phpBB/language/en/ucp.php

```diff
@@ -332,6 +332,7 @@
 	'NOTIFICATION_TYPE_GROUP_REQUEST'					=> 'Someone requests to join a group you lead',
 	'NOTIFICATION_TYPE_FORUM'							=> 'Someone replies to a topic in a forum to which you are subscribed',
 	'NOTIFICATION_TYPE_IN_MODERATION_QUEUE'				=> 'A post or topic needs approval',
+	'NOTIFICATION_TYPE_MENTION'							=> 'Someone mentions you in a post',
```

Comment:
Shouldn't this be in past tense?

Reply:
I think this is actually in line with the other descriptions:
![image](https://user-images.githubusercontent.com/394418/119266115-c9106100-bbe9-11eb-965a-1657a2a82ade.png)